.github/workflows/on-push.yml

```diff
@@ -16,6 +16,7 @@ concurrency:
 jobs:
   validate-changelog:
     runs-on: ubuntu-latest
+    if: github.event.pull_request.base.ref == 'master'
```

Comment:
so this only limits us to pull requests to master?

Reply:
it would be better to use `ref_name`

from github docs:
```
The short ref name of the branch or tag that triggered the workflow run. This value matches the branch or tag name shown on GitHub.
```